zio-interop-cats-tests/shared/src/test/scala/zio/interop/CatsSpecBase.scala

```diff
@@ -64,9 +62,12 @@ private[zio] trait CatsSpecBase
         case infinite: Infinite     =>
           ZIO.dieMessage(s"Unexpected infinite duration $infinite passed to Ticker")
       }
+
+      def scheduler(implicit trace: ZTraceElement): UIO[Scheduler] =
+        ???
```

Comment:
Not needed?

Reply:
This is part of the interface of `Clock` in ZIO 2.0 so we need to honor the contract though like some of the other methods above it will never be called in tests so we don't actually have to implement it.